spec/controllers/conferences_controller_spec.rb

```diff
@@ -54,4 +54,16 @@
       end
     end
   end
+
+  describe 'POST create' do
```

Comment:
Do you think you can come up with a positive example as well? At best testing both, the xhr and the http request.

Reply:
I tried using xhr post, but I kept getting a no method error... Maybe this should be a separate issue?

Reply:
Can you post a snipped of the code you tried? Background: the format changed from Rails 4 to 5 -> xhr is now simply a parameter of the `post` method:
```ruby
post :create, params: params, xhr: true
```

Reply:
Thanks! It worked!